### PR TITLE
@ #478 | should setup dev mode for docs on k8s

### PR DESCRIPTION
**Pull request by hoatle**
connect #478 

requires this PR: https://github.com/teracyhq-incubator/teracy-dev-k8s/pull/82

### CODE REVIEW

**Comment by hoatle** [comment]
@datphan make sure to verify that this works on macOS, Linux, Windows

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
+# teracy-dev docs
+
+Follow this guide to work on the docs
```
Comment (hoavt) [inline docs/README.md]: Should have the full stop at the end of the sentence.

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+## Prerequisites
+
+- `docker-registry` must be available by following https://github.com/teracyhq-incubator/teracy-dev-k8s/blob/develop/docs/docker-registry.md
```
Comment (hoavt) [inline docs/README.md]: Should have the full stop at the end of the sentence.

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+
+- `docker`, `kubectl`, `helm`, `skaffold` must be installed on your host machine and make sure it works
+  with the k8s cluster
```
Comment (hoavt) [inline docs/README.md]: Should have the full stop at the end of the sentence.

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+==> v1beta1/Ingress
+NAME                 HOSTS                      ADDRESS  PORTS  AGE
+teracy-dev-docs-dev  dev.docs.teracy-dev.local  80, 443  0s
+
+==> v1/Pod(related)
+NAME                                  READY  STATUS             RESTARTS  AGE
+teracy-dev-docs-dev-5ccfc8b869-2l7mk  0/1    ContainerCreating  0         0s
+
+
+NOTES:
```
Comment (hoavt) [inline docs/README.md:228]: is this section in console?

File: docs/README.md [hunk 5]
```diff
@@ -0,0 +1,250 @@
...
+1. Get the application URL by running these commands:
+  https://dev.docs.teracy-dev.local/
+
+Deploy complete in 1.669862476s
+Watching for changes every 1s...
+```
+
+- Configure the domain alias by editing the `k8s-dev/workspace/teracy-dev-entry/config_override.yaml`
+with the following configuration:
```
Comment (hoavt) [inline docs/README.md]: the following configuration --> the configuration below:

("following" is used to many times :))

**Review by hoavt — COMMENTED** [review]
@hoatle Please check my comments.

File: docs/README.md [hunk 6]
```diff
@@ -0,0 +1,250 @@
...
+$ skaffold config set default-repo registry.k8s.local
+```
+
+## Skaffold dev mode with kubectl
+
+- Execute the following commands:
+
+```bash
+$ cd docs
+$ skaffold dev
```
Comment (datphan) [inline docs/README.md:38]: Not work with `Docker for Desktop` and our k8s vm on Windows with the same reason https://github.com/teracyhq-incubator/teracy-dev-k8s/pull/82/files#r264996507

File: docs/README.md (hunk 4 above)
Comment (hoatle) [reply docs/README.md:228]: yes

File: docs/README.md (hunk 6 above)
Comment (hoatle) [reply docs/README.md:38]: we're going to use `docker daemon remote access` so that it should work on Windows then.

**Review by hoatle — COMMENTED** [review]
comments applied

**Comment by hoatle** [comment]
PR is updated to get it to work locally or remotely with k8s clusters.

**Review by hieptranquoc — APPROVED** [review]

**Review by datphan — APPROVED** [review]

**Pull request merged**